src/chunk.js

```diff
@@ -151,7 +152,7 @@ class Chunks {
     } catch (e) {
       // We run early in init. All errors should show the doc.
       makeBodyVisible(self.document);
-      throw e;
+      rethrowAsync(e);
```

Comment:
what if `makeBodyVisible(self.document);` throws? I suspect it won't get reported for the same reason `throw e` did not get reported.